### PR TITLE
cmd: refactor commands to partially live in `Service::Cmd::x`

### PR DESCRIPTION
**Pull request by SMillerDev**
Part one of the refactoring, but to avoid changing the whole flow in one PR I'm posting this already.

### CODE REVIEW

File: lib/service/commands/cleanup.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Service
+  module Commands
+    module Cleanup
+      module_function
+
+      def run
+        Service::ServicesCli.cleanup
```
Comment (MikeMcQuaid) [inline lib/service/commands/cleanup.rb:9]: Is the plan to move a bunch of the logic into this method?

File: lib/service/unit.rb [hunk 2]
```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+# Wrapper for a formula to handle service-related stuff like parsing and
+# generating the plist file.
+module Service
+  class Unit
```
Comment (MikeMcQuaid) [inline lib/service/unit.rb:6]: Not huge on this name. Any other suggestions?

Comment (SMillerDev) [reply lib/service/unit.rb:6]: I had `Service::Service` before, but that's not much better.

File: lib/service/commands/cleanup.rb (hunk 1 above)
Comment (SMillerDev) [reply lib/service/commands/cleanup.rb:9]: For cleanup I doubt there would be much. But I expect restart to eventually just be `start && stop`

File: lib/service/unit.rb (hunk 2 above)
Comment (MikeMcQuaid) [reply lib/service/unit.rb:6]: `Service::Definition` or `Service::FormulaWrapper` or something?

File: lib/service/unit.rb [hunk 3]
```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+# Wrapper for a formula to handle service-related stuff like parsing and
+# generating the plist file.
```
Comment (MikeMcQuaid) [inline lib/service/unit.rb:4]: not just `the plist file` I guess now?

File: lib/service/commands/cleanup.rb (hunk 1 above)
Comment (MikeMcQuaid) [reply lib/service/commands/cleanup.rb:9]: Yeh, I guess I'm thinking right now "why not just call `Service::ServicesCli.cleanup` directly?". I'd presume "that logic will be moved here" but if not: I'm not sure about this refactoring.

Comment (SMillerDev) [reply lib/service/commands/cleanup.rb:9]: Yeah, I'd like to move as much of the shared logic there as possible. I believe that's also how bundle does it right?

File: lib/service/formula_wrapper.rb [hunk 4]
```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+# Wrapper for a formula to handle service-related stuff like parsing and
+# generating the service/plist files.
+module Service
+  class FormulaWrapper
```
Comment (MikeMcQuaid) [inline lib/service/formula_wrapper.rb:6]: 👍🏻 but seems there's still a couple of `Unit` references around.

**Review by MikeMcQuaid — APPROVED** [review]
👍🏻 once final issues fixed and 💚

**Pull request merged**